webpack.config.js

```diff
@@ -1,156 +1,128 @@
+      'app/searchResults/searchResults.component.js',
+      'app/profile/yourGiving/yourGiving.component.js',
+      'app/profile/profile.component.js',
+      'app/profile/receipts/receipts.component.js',
+      'app/profile/payment-methods/payment-methods.component.js',
+      'app/designationEditor/designationEditor.component.js'
+    ],
+    'branded-checkout': [
+      'app/branded/branded-checkout.component.js',
+      'assets/scss/branded-checkout.scss'
+    ],
+    give: 'assets/scss/styles.scss',
+    // Add `main` entry on development
+    ...(isBuild ? {} : { main: 'app/main/main.component.js' })
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist')
+  },
+  devtool: 'source-map',
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].min.css',
+      disable: !isBuild
+    }),
+    new CopyWebpackPlugin([
+      {
+        context: 'src',
+        from: '**/*.+(eot|png|svg|ttf|woff)',
+        to: '[path][name].[ext]'
+      },
+      'unsupportedBrowser.html',
+      'branded-checkout.html'
+    ]),
+    new webpack.EnvironmentPlugin({
+      TRAVIS_COMMIT: 'development'
+    }),
+    new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en/)
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: [{
+          loader: 'babel-loader',
+          options: {
+            presets: [['@babel/preset-env', { modules: false }]],
+            plugins: [
+              '@babel/plugin-transform-runtime',
+              'angularjs-annotate'
             ]
-        },
-        // inline css for components with the component's js
-        {
-          test: /\.(scss|css)$/,
-          exclude: path.resolve(__dirname, './src/assets'),
-          use: [
-            'style-loader',
-            'css-loader?sourceMap',
-            'sass-loader?sourceMap'
-          ]
-        }
-      ]
-    },
-    resolve: {
-      modules: [path.resolve(__dirname, 'src'), 'node_modules']
+          }
+        }]
+      },
+      {
+        test: /\.html$/,
+        use: [
+          `ngtemplate-loader?relativeTo=${path.resolve(__dirname, './src')}/`,
+          'html-loader'
+        ]
+      },
+      {
+        test: /\.s?[ac]ss$/,
+        include: path.resolve(__dirname, './src/assets'),
+        use: [
+          MiniCssExtractPlugin.loader,
+          'css-loader?-url',
+          'sass-loader?sourceMap'
+        ]
+      },
+      {
+        test: /\.s?[ac]ss$/,
+        exclude: path.resolve(__dirname, './src/assets'),
+        use: [
+          'style-loader',
+          'css-loader?sourceMap',
+          'sass-loader?sourceMap'
+        ]
+      }
+    ]
+  },
+  resolve: {
+    modules: [path.resolve(__dirname, 'src'), 'node_modules']
+  },
+  devServer: {
+    https: true,
+    historyApiFallback: {
+      rewrites: [{ from: /\/(?!test-release).+\.html/, to: '/index.html' }]
     },
-    devtool: 'source-map',
```

Comment:
Was there a reason you removed this? Sourcemaps are nice for Rollbar and debugging production code. Although Rollbar doesn't seem to be picking them up. It's working with the Chrome dev tools on prod fine.

I'm not up to date on Webpack defaults. Maybe the default is good now.